hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/SignedChunksInputStream.java

```diff
@@ -23,19 +23,26 @@
 import java.util.regex.Pattern;
 
 /**
- * Input stream implementation to read body with chunked signatures.
- * <p>
- * see: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
+ * Input stream implementation to read body with chunked signatures. This should also work
+ * with the chunked payloads with trailer.
+ *
+ * Note that there are no actual chunk signature verification taking place. The InputStream only
+ * returns the actual chunk payload from chunked signatures format.
```

Comment:
I am not familiar with AWS Signature Version 4 but does it mean there'll be a follow up task to verify chunk signatures?

Reply:
Thanks for the review. Raised https://issues.apache.org/jira/browse/HDDS-12542.